plugins/module_utils/deps.py

```diff
@@ -50,7 +50,7 @@ def message(self):
     def failed(self):
         return self.state == 1
 
-    def verify(self, module):
+    def validate(self, module):
```

Comment:
This is a breaking change. You should add a wrapper with the old name that calls the new one, and also `module.deprecate()` to announce that some code needs to be changed :)

Reply:
This is inside the classe `_Dependency`, no one should be using that. No object of this class is returned to code outside this file.